.github/workflows/main.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Does this result in all the CI actions being run twice when a new commit is pushed and a PR opened? I thought I had that experience before...